lib/features/composer/data/network/composer_api.dart

```diff
@@ -0,0 +1,28 @@
+
+import 'dart:io';
+
+import 'package:core/core.dart';
+import 'package:dio/dio.dart';
+import 'package:model/model.dart';
+
+class ComposerAPI {
+
+  final DioClient _dioClient;
+
+  ComposerAPI(this._dioClient);
+
+  Future<UploadResponse> uploadAttachment(UploadRequest uploadRequest) async {
+    final resultJson = await _dioClient.post(
+        Uri.decodeFull(uploadRequest.uploadUrl.toString()),
+        options: Options(headers: _buildHeaderRequestParam(uploadRequest.accountRequest.basicAuth())),
```

Comment:
we still need the `basicAuth` here when we have the interceptor for BasicAuth ?

Reply:
OK. I forgot this.